addon/components/base/bs-modal-simple.js

```diff
@@ -54,12 +54,12 @@ import layout from 'ember-bootstrap/templates/components/bs-modal-simple';
  then give it the HTML5 autofocus attribute:
 
  ```hbs
- {{#bs-modal-simple title="Form Example" closeTitle="Cancel" submitTitle="Ok"}}
-   {{#bs-form onSubmit=(action "submit") model=this as |form|}}
-     {{form.element controlType="text" label="first name" property="firstname" autofocus=true}}
-     {{form.element controlType="text" label="last name" property="lastname"}}
-   {{/bs-form}}
- {{/bs-modal-simple}}
+ <BsModalSimple @title="Form Example" @closeTitle="Cancel" @submitTitle="Ok">
+  <BsForm @model={{this}} @onSubmit={{action "submit"}} as |Form|>
+    <Form.element @controlType="text" @label="first name" @property="firstname" @autofocus={{true}}/>
```

Comment:
Before there were spaces between closing curlies `}}` and self-closing element `/>`. Not quite sure what should be considered best practice and if `ember-template-lint` comes with a linting rule for that one. But it should be the same everywhere IMO.

Reply:
Not sure, for now i'll just remove them